spec/client/App.spec.jsx

```diff
@@ -1,8 +1,13 @@
 import React from 'react';
 import renderer from 'react-test-renderer';
+import { shallow } from 'enzyme';
```

Comment:
Is there a reason new modules and files were added to this file and then not used?

Reply:
Nope, just accidental leftovers. Probably from moving tests to FilterButton spec files IIRC. Just updated it to remove those modules.